.env.example

```diff
@@ -0,0 +1,3 @@
+IG_USERNAME = null
+IG_PASSWORD = null
+IG_PROXY = null
```

Comment:
Setting `IG_PROXY` to `"null"` will always set the proxy. The proxy should be a opt-in.
```suggestion
# Uncomment this to set a proxy
# IG_PROXY=
```